modules/enhancer.js

```diff
@@ -46,8 +46,16 @@ var enhanceWithRadium = function (ComposedComponent: constructor): constructor {
 
   // Class inheritance uses Object.create and because of __proto__ issues
   // with IE <10 any static properties of the superclass aren't inherited and
-  // so need to be manually populated
+  // so need to be manually populated. We only want to go over that object's
+  // non-enumerable properties, so use Object.keys.
   // See http://babeljs.io/docs/advanced/caveats/#classes-10-and-below-
+  Object.keys(ComposedComponent).forEach(key => {
```

Comment:
:+1: done and it fixes the test that was then failing because we weren't copying across static methods defined using `static` keyword within class definitions.